R/pkg/R/generics.R

```diff
@@ -695,10 +695,6 @@ setGeneric("desc", function(x) { standardGeneric("desc") })
 
 #' @rdname column
 #' @export
-setGeneric("endsWith", function(x, ...) { standardGeneric("endsWith") })
```

Comment:
you shouldn't need to remove it - I think if they are updated to match the signature of the existing generic (in newer R 3.3.0) it should work both in the case of R 3.3.0 and R 3.2.0.
With regards to the comment on R version check - it should not be necessary also.
@shivaram